### DISCUSSION TITLE
How to access assetPath in theme css?

### DISCUSSION THREAD

**Discussion by fooyoo** [TriliumNext/Trilium#3313] [Q&A] [answered]
After upgrade to 0.57.0-beta, in my theme, the menu icon disappears. I noticed the original icon path is changed:
from 
    background-image: url("/images/icon-grey.png");
to
    background-image: url("/assets/v0.57.0-beta/images/icon-grey.png");

I don't want to change the path every times after upgrading software, is any variable or better way for that?

**Comment by zadam** [comment, marked as answer]
You can use `/assets/vX/images/...` which is an alias for such cases.

**Reply by fooyoo** [reply]
That's exactly what I want! Thank you Zadam.
